src/core/blocks/base_block.c

```diff
@@ -0,0 +1,59 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "base_block.h"
+
+char * ser_BlockHeader(BlockHeader *block_header, char* dest){
+    memcpy(dest, &(block_header->timestamp), sizeof(block_header->timestamp));
+
+    char * all_tx = dest+sizeof(block_header->timestamp);
```

Comment:
I'd also like to be consistent for `*` placement. I believe convention is `int *name` for variables, and `int* func_name` for functions? Open to whatever but we should be consistent

Reply:
yes lets do ```int *name``` for variables as well as ```int *function```